src/network/networkd-address-label.h

```diff
@@ -19,6 +19,7 @@ typedef struct NetworkConfigSection NetworkConfigSection;
 struct AddressLabel {
         Network *network;
         NetworkConfigSection *section;
+        bool section_is_invalid;
```

Comment:
shouldn't this bool move into NetworkConfigSection? After all it's a property of that, and pretty much all things configured with sections will have a bool like that, no? And even if they don't, it shouldn't matter much having that extra bool in there...